lib/redis_test.rb

```diff
@@ -54,7 +59,9 @@ def start
         "loglevel"      => loglevel,
         "databases"     => 16
       }.map { |k, v| "#{k} #{v}" }.join('\n')
-      `echo '#{redis_options}' | redis-server -`
+
+      `echo '#{redis_options}' | '#{redis_server_command}' -`
```

Comment:
Could you use Shellwords.escape instead of naive quoting? I will accept that pull request then